test/assertion.js

```diff
@@ -127,11 +127,16 @@ describe('Assertion', function(){
       });
     });
 
-    it('should map ecommerce events', function(){
-      segment.mapper.completedOrder = function(t){ return t.properties(); };
+    it('should map actionObject ecommerce events', function(){
+      segment.mapper.orderCompleted = function(t){ return t.properties(); };
```

Comment:
confused by this test. don't we want one of these to test completedOrder and the other one orderCompleted?